draft-ietf-ppm-dap.md

```diff
@@ -1843,7 +1837,7 @@ struct {
 struct {
   opaque agg_param<0..2^32-1>;
   PartialBatchSelector part_batch_selector;
-  PrepareInit prepare_inits<0..2^32-1>;
+  PrepareInit prepare_inits<46..2^32-1>;
```

Comment:
I got 44:
```
﻿﻿PrreapreInit {
  ReportShare {
    ReportMetadata {
      ReportID 16
      Time 8
      Extensions 2 + 0
    }
    opaque 4 + 0
    HpkeCiphertext {
      HpkeConfigID 1
      opaque 2 + 1
      opaque 4 + 1
    }
  }
  opaque 4 + 1
}
```

Reply:
You are correct, I misinterpreted the public extensions and public share fields as being at least 1 byte long, but they're both allowed to be empty.

Reply:
Will update, then merge.

Reply:
You got there first!